### PR TITLE
Product listing and filter UI

### PR DESCRIPTION
**Pull request by ScoobyDubeyDo**
- Add Filters UI
 > price slider
 Types checkboxes
 rating radio
 sorting radio 
- Products listing section
- And fixed a few minor bugs

### CODE REVIEW

**Review by xsaurabhsuthar — APPROVED** [review]
- Good folder structure.
- Code looks clean.
- Try to remove the commented code.
- You should not keep assets or any static files in the src dir, move them to public dir.
Good to merge 🚀🚀

File: src/pages/Products/components/FilterBox/components/FilterClear/filterClear.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.cart-price-pair {
```
Comment (shraddha-amber) [inline src/pages/Products/components/FilterBox/components/FilterClear/filterClear.css:1]: repeated css same as priceSlider.css

File: src/pages/Products/components/FilterBox/components/FilterClear/FilterClear.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import "./filterClear.css";
+
+export const FilterClear = () => {
+    return (
+        <div className="cart-price-pair border-bottom">
+            <h1 className="heading-4">Filter</h1>
+            <button
+                className="btn-link heading-6"
+                // onClick={() => dispatch({ type: "CLEAR" })}
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/FilterClear/FilterClear.jsx:9]: Remove commented code

File: src/pages/Products/components/FilterBox/components/PriceSlider/PriceSlider.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <div className="cart-price-pair">
+                <span className="heading-6">240</span>
+                <span className="heading-6">{selectedPrice}</span>
+            </div>
+            <input
+                type="range"
+                min="240"
+                max="1599"
+                value={selectedPrice}
+                // onChange={(e) =>
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/PriceSlider/PriceSlider.jsx:16]: remove commented code

File: src/pages/Products/components/FilterBox/components/RatingRadio/RatingRadio.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+export const RatingRadio = ({ starsArray }) => {
+    return (
+        <>
+            <h3 className="heading-5">Rating</h3>
+            <span>
+                {starsArray.map((star) => (
+                    <label key={star} htmlFor={star} className="star-display">
+                        <input
+                            // onChange={(e) => {
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/RatingRadio/RatingRadio.jsx:12]: remove commented code

File: src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                            value={sort}
+                            // checked={state.sortStrategy === sort}
+                            // onChange={(e) =>
+                            //     dispatch({
+                            //         type: "SORT_BY_PRICE",
+                            //         payload: { sortWay: e.target.value },
+                            //     })
+                            // }
+                        />
+                        {` Price -
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx:25]: This logic code can be removed from here. You can make a separate function for this functionality and call it here.

File: src/pages/Products/components/FilterBox/components/TypesCheckbox/TypesCheckbox.jsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+                    return (
+                        <label
+                            key={type}
+                            htmlFor={`${type.toLowerCase()}`}
+                            className="rootShoot-margin-center text-body-lg"
+                        >
+                            <input
+                                id={`${type.toLowerCase()}`}
+                                name="types"
+                                // checked={state.selectedTypes.includes(
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/TypesCheckbox/TypesCheckbox.jsx:16]: Remove commented code.

File: src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx (hunk 5 above)
Comment (shraddha-amber) [inline src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx:25]: suggestion: might separate the logic in a function

File: src/pages/Products/components/FilterBox/components/TypesCheckbox/TypesCheckbox.jsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+                                //               type: "TYPES",
+                                //               payload: {
+                                //                   typeOfProd: e.target.value,
+                                //                   isChecked: false,
+                                //               },
+                                //           })
+                                // }
+                            />
+                            {`  ${
+                                type.charAt(0).toUpperCase() + type.slice(1)
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/TypesCheckbox/TypesCheckbox.jsx:42]: create a different function like getLable() to put this functionality in there, You are reusing the same logic at multiple places.

File: src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+                        key={sort}
+                        htmlFor={sort}
+                        className="rootShoot-margin-center text-body-lg"
+                    >
+                        <input
+                            name="sort"
+                            id={sort}
+                            type="radio"
+                            value={sort}
+                            // checked={state.sortStrategy === sort}
```
Comment (shobit1337) [inline src/pages/Products/components/FilterBox/components/SortbyRadio/SortbyRadio.jsx:17]: remove commented code

**Review by shobit1337 — CHANGES_REQUESTED** [review]
Suggested some changes.
- There are commented codes at multiple places try to remove them before pushing them. 
- Try to avoid putting logic in your JSX, and put it in a separate function.

The rest of the things looks awesome. Great work @aman-can 😄👍

**Pull request merged**